provisions/roles/jenkins/slave/tasks/setup_worker.yml

```diff
@@ -82,26 +82,31 @@
 
 - name: Pull Linter image
   docker_image:
-      name: registry.centos.org/pipeline-images/dockerfile-lint
+      name: registry.centos.org/pipeline-images/dockerfile-lint-2
       state: present
       force: yes
   tags:
       - application
 
-- name: Copy Dockerfile linter worker
-  copy:
-      src: ../../scripts/cccp-dockerfile-lint-worker.service
-      dest: /etc/systemd/system/cccp-dockerfile-lint-worker.service
-  tags:
-      - application
-
-- name: Reload Dockerfile linter worker systemd service
-  sudo: yes
-  systemd: name=cccp-dockerfile-lint-worker daemon_reload=yes
+- name: Pull Dockerfile Lint worker image
```

Comment:
Well need `sudo` here,

Reply:
Have we ever done a provisioning as a non-root user on the local or remote system? If yes, I'd add `sudo: yes` to it; otherwise I won't.

Reply:
yes.